modules/ROOT/pages/resource-usage.adoc

```diff
@@ -0,0 +1,35 @@
+= View Runtime Fabric Status and Resources
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can use Anypoint Runtime Manager to monitor the resource usage and health of your Anypoint Runtime Fabric clusters.
+
+== Resource Usage 
+
+To view the resource and vCPU usage:
+
+. Navigate to Runtime Manager and select *Runtime Fabric*.
+. Click in the area between a Runtime Fabric name and its status to display resource and status information for that Runtime Fabric.
+. Click the *Manage* button to open the management page.
++
+Information that is displayed includes version, number of controller, worker, and ingress (internal load balancer) nodes, and  aggregated vCPU core usage.
+
+== Health Details
+For Runtime Fabric version 1.5 or later:
+
+. Navigate to Runtime Manager and select *Runtime Fabric*.
+. Select the name of your Runtime Fabric to open the management page.
+. Select *Health Details*.
++
+The *Health Details* section includes a *Suitable for Production Workloads* section that indicates whether your Runtime Fabric cluster configuration meets production environment requirements, as shown in the following example:
++
```

Comment:
Should this be formatted like example syntax?

Reply:
Would it be an improvement if we enumerated the options the user sees in this section (production, development-only, not supported), and explain this classification is driven by the installation requirements?

Debating whether it'll add confusion if we only show an example for a production-suitable Runtime Fabric for a user who's seeing another option.